x-pack/plugins/security_solution/server/endpoint/services/actions/action_files.ts

```diff
@@ -84,9 +85,17 @@ export const getFileInfo = async (
   fileId: string
 ): Promise<UploadedFileInfo> => {
   try {
-    const fileClient = getFileClient(esClient, logger);
-    const file = await fileClient.get({ id: fileId });
-    const { name, id, mimeType, size, status, created } = file.data;
+    const { _id: id, _source: fileDoc } = await esClient.get<FileUploadMetadata>({
```

Comment:
So.... Turns out that the Files plugin `FileClient.get()` does not actually return the entire File metadata document - only a subset of properties from the `file: {}` object. So instead, I switched this up to use ES to get the file record out.